examples/scripts/ct_abel.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This file is part of the SCICO package. Details of the copyright
+# and user license can be found in the 'LICENSE.txt' file distributed
+# with the package.
+
+r"""
+Abel Transform Demo
+===================
+
+This is a placeholder for more detailed docs."
+"""
+
+import numpy as np
+
+import jax
+
+import abel
+
+import scico.numpy as snp
+from scico import functional, linop, loss, plot
+from scico.linop.abel import AbelProjector
+from scico.optimize.admm import ADMM, LinearSubproblemSolver
+from scico.util import device_info
+
+
+def dist_map_2D(img_shape, center=None):
+
```

Comment:
Functions that are defined in the other example scripts have brief (i.e. without formal Args/Returns sections) docstrings explaining their purpose.

Reply:
Fixed